lighthouse-core/gather/gatherers/css-usage.js

```diff
@@ -44,11 +45,6 @@ class CSSUsage extends Gatherer {
       return driver.sendCommand('CSS.disable')
         .then(_ => driver.sendCommand('DOM.disable'))
         .then(_ => results.ruleUsage);
-    }).catch(err => {
```

Comment:
The `afterPass` won't be called when the recoverable error is thrown right? Otherwise we'll fail on stable Chrome again